app/resources/views/components/session-display.blade.php

```diff
@@ -5,13 +5,16 @@
 ])
 
 
- @if ($currentSession)
+@if ($currentSession)
 
     <div {{ $attributes->bem('session-display') }}>
         {{ __('components.session-display.current', [
             'location' => $currentSession->location->label,
         ]) }}
 
+        {!! __('components.session-display.agenda', [
+            'agenda' => $currentSession->agendaLink]) !!}</br>
```

Comment:
While the naming convention is to use camel case for method names, attributes are usually referred to using snake case.

I think Laravel is smart enough to call the `getAgendaLinkAttribute` accessor for the attribute `agenda_link`, but I’m not 100% sure.

```suggestion
            'agenda' => $currentSession->agenda_link]) !!}</br>
```

Reply:
My test is still green, so I assume Laravel is indeed smart enough. Nice, I did not know this!